README.md

```diff
@@ -18,10 +18,10 @@ Project _VERAISON_: **VER**ific**A**t**I**on of atte**S**tati**ON**
+Attestation is the system by which an entity produces evidence about itself that another party can use to evaluate the trustworthiness of that entity. A critical part of establishing trust based on that evidence is Verifying what is presented as evidence against a set of known elements to ensure it meets an established policy to prove trustworthiness. The act of Verification will often require comparing evidence against reference values or checking cryptographic signing by a known trust anchors. This is a complex task to be performed by an arbitrary user of a system and it is often delegated to a trusted Verification Service instead. The nature of the Service will depend upon specific needs - it may be a local software component or it may reside as a network or internet accessible component. The actual nature of the Service will depend upon the actual deployment. The Verification Service will need up to date data for the process of appraising the evidence presented to it. This must be obtained from authoritative sources, like the supply chain, which normally implies establishing business relationships between the Verification Service and those sources.  
 
 ## What part does Veraison play in this?
-Given the above, it can be challenging to build just one Verification Service solution which can address all deployments for a technology that needs to produce Attestation reports to prove its trustworthiness. If that then implies that each deployment needs a custom service, there is a significant software barrier and hence cost of entry to establishing a system that can be used in a secure manner. Veraison aims to provide consistency and convenience to solving this problem by building the software components that can be used to build Attestation Verification Services. The Veraison project will also build some reference implementations in order to prove the integration principles of those components and to provide a convenient starting point for some implementations.
+Given the above, it can be challenging to build just one Verification Service solution which can address all types of deployments for a technology that needs to produce Attestation reports to prove its trustworthiness. It then implies that each deployment needs a custom service, however there is a significant software barrier and hence cost of entry to establishing such a system to be used in a secure manner. Veraison aims to provide consistency and convenience to solving this problem by building the software components that can be used to build Attestation Verification Services. The Veraison project will also build some reference implementations in order to prove the integration principles of those components and to provide a convenient starting point for some implementations.
```

Comment:
A suggestion instead of your 'however' change:   s/however there is/which creates/

It then implies that each deployment needs a custom service, which creates a a significant software barrier...

Reply:
ok, this is also fine, I will change it!